images/bigquery/Dockerfile

```diff
@@ -14,20 +14,19 @@
 
 # Includes gcloud (for bq and gsutil), python influxdb client (requires pip), python yaml parser, and jq.
 
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y \
     git \
     jq \
     wget \
-    python \
```

Comment:
I missed this in the initial review. I am fairly certain that `python2` is still required in environment by Google Cloud SDK ([ref](https://cloud.google.com/python/setup#installing_the_cloud_sdk))

Reply:
good question... yes, this builds https://gist.github.com/spiffxp/9410a8a093f71b53f4e5da3aafec6d55#file-docker-build-bigquery-txt-L1681

Reply:
🤔. Could result in a runtime error. Honestly not sure how this image is invoked. But since it calls `runner` which call `bootstrap.py` (which requires `python`2) should be included.

Reply:
This sent me down the rabbit hole of trying to use `pj-on-kind.sh`.  Which has taught me some things, but I waved off for now, and ran some commands in a docker container manually.  `bootstrap.py` was happy, `bq` was not. `python` is back in

Reply:
Good to know exactly why it is needed. I vaguely recall the same problem when I upgrading [`triage`](https://github.com/kubernetes/test-infra/pull/13380/files#diff-a4471eb18fe771121327c51c399cbdd3R17-R18).